README.md

```diff
@@ -116,10 +116,10 @@ This allows you to hack on Caddy core (and optionally plug in extra modules at t
 
 ---
 
-You may embed directories into the Caddy executable and serve them from the `embedded` filesystem module:
+If `--embed` is used without an alias prefix, the contents of the source directory are written directly into the root directory of the embedded filesystem within the Caddy executable. The contents of multiple unaliased source directories will be merged together:
 
-```
-$ xcaddy build --embed foo:./sites/foo --embed bar:./sites/bar
+```sh
```

Comment:
Not sure if this is actually what we want, given that most of the content is not shell commands, but rather some output (that happens to be a Caddyfile). Unless the highlighter is smart enough to parse the difference? (Same below)

Reply:
I totally agree and had actually meant to remove this before submitting, but just noticed that my push to my fork timed out and I didn't notice before I hit the button 😭  I'll send a followup in a second